src/vs/workbench/contrib/debug/node/telemetryApp.ts

```diff
@@ -7,7 +7,7 @@ import { Server } from 'vs/base/parts/ipc/node/ipc.cp';
 import { TelemetryAppenderChannel } from 'vs/platform/telemetry/common/telemetryIpc';
 import { OneDataSystemAppender } from 'vs/platform/telemetry/node/1dsAppender';
 
-const appender = new OneDataSystemAppender(undefined, process.argv[2], JSON.parse(process.argv[3]), process.argv[4]);
+const appender = new OneDataSystemAppender(false, process.argv[2], JSON.parse(process.argv[3]), process.argv[4]);
```

Comment:
does it make sense for us to get the actual value here?

Reply:
I don't believe we have access to the proper services here

Reply:
You can pass the value here https://insiders.vscode.dev/github/microsoft/vscode/blob/189950f732aad4ab864e6b68efce5d25bfc9a8d9/src/vs/platform/telemetry/node/customEndpointTelemetryService.ts#L29-L48